conflictmonitor/README.md

```diff
@@ -0,0 +1,7 @@
+# JPO-Conflictmonitor Components
```

Comment:
Is this directory being used in this project? Or just a convenience for when you spin up an associated conflict monitor?

Reply:
These are necessary for running the conflictmonitor and conflictvisualizer together, as this contains the kafka connect config. Without using a submodule, I am not sure how else to include these files. They are sufficiently complex that it would be difficult to run a successful local development system without them

Reply:
Let's expand on this README a bit to explain how these files are used, and why they are necessary. Currently we just mention there's "useful configuration files" but this makes it seem like they are necessary to run the application properly. We could punt and add this as a backlog item as well.